packages/autocomplete-plugin-recent-searches/src/index.ts

```diff
@@ -0,0 +1 @@
+export { createPlugin as createRecentSearchesPlugin } from './createPlugin';
```

Comment:
We can call the function `createRecentSearchesPlugin` directly in our code. It'll be easier to maintain.

Reply:
done https://github.com/algolia/autocomplete.js/pull/316/commits/90fc79061c0beeb95cceedd446d8f1e2445a8b21